### PR TITLE
[NOSQUASH] use our irrlicht fork on docker server

### PR DESCRIPTION
**Pull request by nerzhul**
Add compact, short information about your PR for easier understanding:

* Fix alpine version to be up-to-date
* Build our irrlicht version & use it

## To do

This PR is Ready for Review.
## How to test

`docker build . -t minetest:latest && docker run minetest:latest`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -36,6 +37,14 @@ RUN git clone --recursive https://github.com/jupp0r/prometheus-cpp/ && \
 	make -j2 && \
 	make install
 
+RUN git clone --depth=1 https://github.com/minetest/irrlicht/ -b ${IRRLICHT_VERSION} && \
+	mkdir irrlicht/build && \
```
Comment (sfan5) [inline Dockerfile:41]: `cp -r irrlicht/include /usr/include/irrlichtmt` would be enough here, the server doesn't need irrlicht built
(you can drop bzip2-dev, libpng-dev, jpeg-dev, libxxf86vm-dev, mesa-dev, libogg-dev, libvorbis-dev, openal-soft-dev, freetype-dev then too)

Comment (nerzhul) [reply Dockerfile:41]: i'll try to drop some libs, you are right :D i used the original method at the beginning of this dockerfile. That should massiverly lightweight the image

**Comment by nerzhul** [comment]
works perfectly with your ideas :) great

File: Dockerfile [hunk 2]
```diff
@@ -18,9 +19,7 @@ COPY textures /usr/src/minetest/textures
 
 WORKDIR /usr/src/minetest
 
-RUN apk add --no-cache git build-base irrlicht-dev cmake bzip2-dev libpng-dev \
-		jpeg-dev libxxf86vm-dev mesa-dev sqlite-dev libogg-dev \
-		libvorbis-dev openal-soft-dev curl-dev freetype-dev zlib-dev \
+RUN apk add --no-cache git build-base cmake sqlite-dev openal-soft-dev curl-dev zlib-dev \
```
Comment (sfan5) [inline Dockerfile:22]:
```suggestion
RUN apk add --no-cache git build-base cmake sqlite-dev curl-dev zlib-dev \
```

**Review by sfan5 — APPROVED** [review]
Looks good otherwise.

**Pull request merged**